README.md

```diff
@@ -18,8 +18,8 @@ Pinning specific versions or SHAs is recommended to avoid unplanned upgrades.
 
 ```
 # Start with the latest versions and don't just copy what's here
-octodns==0.9.14
-octodns-googlecloud==0.0.1
+octodns==0.9.21
+octodns-googlecloud==0.0.2
```

Comment:
Doesn't hurt anything to update these, but they're generally just left alone during new releases and it's assumed/hoped that people will pick good versions. Perhaps I should go through and change them all to placeholders to make that more clear. Don't need to make that change as part of this PR. I prefer 1-change PRs.